src/bot/root_pythia_cogs.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from discord.ext import commands, tasks
+
+NAME = "RootPythiaCommands"
+
+class RootPythiaCommands(commands.Cog, name=NAME):
+    """Define the commands that the bot will respond to, prefixed with the `command_prefix` defined at the bot init"""
+
+    def __init__(self, bot, dbmanager):
+        self.bot = bot
+        self.logger = logging.getLogger(__name__)
+        self.dbmanager = dbmanager
+
+        self.check_new_solves.start()
+
+    # FIXME: this decorator won't work...
+    def log_command_call(cmd):
+        async def log_it(self, ctx, *args, **kwargs):
+            self.logger.info("'%s' command triggered by '%s'", ctx.command, ctx.author)
+            await cmd(self, ctx, *args, **kwargs)
+        return log_it
+
+    @commands.command(name='hey')
+    async def hey(self, ctx):
+        await ctx.message.channel.send("hey command works!!\nJust happy to be alive")
+
+    @commands.command(name='ping')
+    async def ping(self, ctx):
+        await ctx.message.channel.send("weird habit... but I guess: pong?...")
+
+    # TODO: add a add_users command that would accept a list of ids
+    @commands.command(name='adduser')
+    async def add_user(self, ctx, idx: int):
+        user = await self.dbmanager.add_user(idx)
+
+        if user is None:
+            await ctx.message.channel.send(f"UserID {idx} already exists in database")
```

Comment:
And if there is an error?

Reply:
Hmmm, I think there isn't a problem here actually, if the DBManger raises an exception it will be passed to the `getuser` command, triggering the `on_error` event and ignored otherwise by discord.py.
We then have to deal withb it in such a `on_error` event

If you really want to deal with it with return value:
Maybe `self.dbmanager.add_user` should return `True` (this can be tested then with `user is True` I think) if user exists and `None` if any error is raised?

Reply:
4319e1e this could be sufficient for now!